BASE/Test/Microsoft.ApplicationInsights.Test/Microsoft.ApplicationInsights.Tests/Channel/TransmissionTest.cs

```diff
@@ -363,6 +366,85 @@ public async Task SendAsyncReturnsCorrectHttpResponseWrapperWithRetryHeaderWhenN
                 }
 
             }
+
+#if NETCOREAPP2_1
+            [TestMethod]
+            public async Task SendAsyncLogsIngestionReponseTimeEventCounter()
+            {
+                var handler = new HandlerForFakeHttpClient
+                {
+                    InnerHandler = new HttpClientHandler(),
+                    OnSendAsync = (req, cancellationToken) =>
+                    {
+                        return Task.FromResult<HttpResponseMessage>(new HttpResponseMessage());
+                    }
+                };
+
+                using (var fakeHttpClient = new HttpClient(handler))
+                {
+                    // Instantiate Transmission with the mock HttpClient
+                    Transmission transmission = new Transmission(testUri, new byte[] { 1, 2, 3, 4, 5 }, fakeHttpClient, string.Empty, string.Empty);
+
+                    using (var listener = new TestEventListener())
+                    {
+                        var eventCounterArguments = new Dictionary<string, string>
+                        {
+                            {"EventCounterIntervalSec", "1"}
+                        };
+
+                        listener.EnableEvents(CoreEventSource.Log, EventLevel.LogAlways, (EventKeywords)AllKeywords, eventCounterArguments);
+
+                        HttpWebResponseWrapper result = await transmission.SendAsync();
+
+                        // VERIFY
+                        // We validate by checking SDK traces.
+                        var allTraces = listener.Messages.ToList();
```

Comment:
Dont we need to wait for atleast 1 sec (as the interval is 1sec) ?

Reply:
Makes Sense.